Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY	.	.
 
 RUN	make "version=$version"
 
-FROM	debian:buster-slim
+FROM	livepeerci/build:latest
```

Comment:
Should we remove the `apt install ffmpeg` below? Or is it quick when its already installed?